src/modules/45-dashboards/pages/DashboardsCommonHeaderPage.tsx

```diff
@@ -39,11 +41,9 @@ const DashboardsHeader: React.FC = () => {
     updateTitle(titleArray)
   }, [breadcrumbs, getString, updateTitle])
 
-  return (
-    <Page.Header
-      title={title}
-      breadcrumbs={<NGBreadcrumbs links={breadcrumbs} />}
-      content={
+  const dashboard_navigation: React.ReactNode | undefined = React.useMemo(() => {
```

Comment:
```suggestion
  const dashboard_navigation: React.ReactNode = React.useMemo(() => {
```

Don't use `undefined` at returntype. Return `null` instead when condition is false.